From dfd99f63f744ecbda0e874505a897d9e26b6adf9 Mon Sep 17 00:00:00 2001
From: User <user@example.com>
Date: Tue, 16 Dec 2025 11:03:34 +0100
Subject: test/automation: remove skipped tests

- Remove caching tests as they tests numba's implementation
- Remove test_ufunc_at_negative_indexes as it hangs in docker
- Remove test_parallel_backend as they hang in docker (spawning multiple processes)
---
 everything.txt => all-tests.txt | 136 --------------------------------
 1 file changed, 136 deletions(-)
 rename everything.txt => all-tests.txt (98%)

diff --git a/everything.txt b/all-tests.txt
similarity index 98%
rename from everything.txt
rename to all-tests.txt
index 7f3b2cb16..129b619be 100644
--- a/everything.txt
+++ b/all-tests.txt
@@ -38,21 +38,6 @@ numba.tests.gdb.test_break_on_symbol.Test.test
 numba.tests.gdb.test_break_on_symbol_version.Test.test
 numba.tests.gdb.test_conditional_breakpoint.Test.test
 numba.tests.gdb.test_pretty_print.Test.test
-numba.tests.npyufunc.test_caching.TestCacheSpecificIssue.test_first_load_cached_gufunc
-numba.tests.npyufunc.test_caching.TestCacheSpecificIssue.test_first_load_cached_ufunc
-numba.tests.npyufunc.test_caching.TestDUfuncCacheTest.test_direct_dufunc_cache
-numba.tests.npyufunc.test_caching.TestDUfuncCacheTest.test_indirect_dufunc_cache
-numba.tests.npyufunc.test_caching.TestGUfuncCacheTest.test_direct_gufunc_cache
-numba.tests.npyufunc.test_caching.TestGUfuncCacheTest.test_direct_gufunc_cache_objmode
-numba.tests.npyufunc.test_caching.TestGUfuncCacheTest.test_direct_gufunc_cache_parallel
-numba.tests.npyufunc.test_caching.TestGUfuncCacheTest.test_filename_prefix
-numba.tests.npyufunc.test_caching.TestGUfuncCacheTest.test_indirect_gufunc_cache
-numba.tests.npyufunc.test_caching.TestGUfuncCacheTest.test_indirect_gufunc_cache_parallel
-numba.tests.npyufunc.test_caching.TestUfuncCacheTest.test_direct_ufunc_cache
-numba.tests.npyufunc.test_caching.TestUfuncCacheTest.test_direct_ufunc_cache_objmode
-numba.tests.npyufunc.test_caching.TestUfuncCacheTest.test_direct_ufunc_cache_parallel
-numba.tests.npyufunc.test_caching.TestUfuncCacheTest.test_indirect_ufunc_cache
-numba.tests.npyufunc.test_caching.TestUfuncCacheTest.test_indirect_ufunc_cache_parallel
 numba.tests.npyufunc.test_dufunc.TestDUFunc.test_frozen
 numba.tests.npyufunc.test_dufunc.TestDUFunc.test_npm_call
 numba.tests.npyufunc.test_dufunc.TestDUFunc.test_npm_call_implicit_output
@@ -78,7 +63,6 @@ numba.tests.npyufunc.test_dufunc.TestDUFuncAt.test_ufunc_at_inner_loop
 numba.tests.npyufunc.test_dufunc.TestDUFuncAt.test_ufunc_at_large
 numba.tests.npyufunc.test_dufunc.TestDUFuncAt.test_ufunc_at_multiD
 numba.tests.npyufunc.test_dufunc.TestDUFuncAt.test_ufunc_at_negative
-numba.tests.npyufunc.test_dufunc.TestDUFuncAt.test_ufunc_at_negative_indexes
 numba.tests.npyufunc.test_dufunc.TestDUFuncAt.test_ufunc_at_no_loop_for_op
 numba.tests.npyufunc.test_dufunc.TestDUFuncAt.test_ufunc_at_not_none_signature
 numba.tests.npyufunc.test_dufunc.TestDUFuncAt.test_ufunc_at_output_casting
@@ -1143,13 +1127,6 @@ numba.tests.test_builtins.TestStrAndReprBuiltin.test_str_object_kwarg
 numba.tests.test_byteflow.TestByteFlowIssues.test_issue_5087
 numba.tests.test_byteflow.TestByteFlowIssues.test_issue_5097
 numba.tests.test_byteflow.TestByteFlowIssues.test_issue_5680
-numba.tests.test_caching.TestCacheLocatorEnvironmentIntegration.test_default_locators
-numba.tests.test_caching.TestCacheLocatorEnvironmentIntegration.test_locators_env_override_custom
-numba.tests.test_caching.TestCacheLocatorEnvironmentIntegration.test_locators_env_override_list
-numba.tests.test_caching.TestCacheLocatorEnvironmentIntegration.test_locators_env_override_single
-numba.tests.test_caching.TestCacheLocatorEnvironmentIntegration.test_locators_env_override_unknown
-numba.tests.test_caching.TestInTreeCacheLocatorFsAgnostic.test_source_stamp_precision
-numba.tests.test_caching.TestInTreeCacheLocatorFsAgnostic.test_timestamp_precision_on_fs
 numba.tests.test_casting.TestCasting.test_0darrayT_to_T
 numba.tests.test_casting.TestCasting.test_array_to_array
 numba.tests.test_casting.TestCasting.test_array_to_scalar
@@ -1233,7 +1210,6 @@ numba.tests.test_closure.TestInlinedClosure.test_inner_function_with_closure_2
 numba.tests.test_closure.TestInlinedClosure.test_inner_function_with_closure_3
 numba.tests.test_closure.TestInlinedClosure.test_issue9222
 numba.tests.test_closure.TestInlinedClosure.test_issue_9577
-numba.tests.test_codegen.JITCPUCodegenTestCase.test_cache_disabled_inspection
 numba.tests.test_codegen.JITCPUCodegenTestCase.test_get_pointer_to_function
 numba.tests.test_codegen.JITCPUCodegenTestCase.test_library_lifetime
 numba.tests.test_codegen.JITCPUCodegenTestCase.test_magic_tuple
@@ -1242,9 +1218,6 @@ numba.tests.test_codegen.JITCPUCodegenTestCase.test_serialize_unserialize_object
 numba.tests.test_codegen.JITCPUCodegenTestCase.test_unserialize_other_process_bitcode
 numba.tests.test_codegen.JITCPUCodegenTestCase.test_unserialize_other_process_object_code
 numba.tests.test_codegen.TestWrappers.test_noinline_on_main_call
-numba.tests.test_compile_cache.TestCompileCache.test_cache
-numba.tests.test_compile_cache.TestCompileCache.test_closures
-numba.tests.test_compile_cache.TestCompileCache.test_error_model
 numba.tests.test_compiler_flags.TestCompilerFlagCachedOverload.test_fastmath_in_overload
 numba.tests.test_compiler_flags.TestCompilerFlags.test_setting_invalid_attribute
 numba.tests.test_compiler_flags.TestFlagMangling.test_demangle
@@ -4209,7 +4182,6 @@ numba.tests.test_parfors.TestParforsMisc.test_issue_9182_recursion_error
 numba.tests.test_parfors.TestParforsMisc.test_issue_9678_build_map
 numba.tests.test_parfors.TestParforsMisc.test_lookup_cycle_detection
 numba.tests.test_parfors.TestParforsMisc.test_no_state_change_in_gufunc_lowering_on_error
-numba.tests.test_parfors.TestParforsMisc.test_no_warn_if_cache_set
 numba.tests.test_parfors.TestParforsMisc.test_oversized_tuple_as_arg_to_kernel
 numba.tests.test_parfors.TestParforsMisc.test_parfor_ufunc_typing
 numba.tests.test_parfors.TestParforsMisc.test_statement_reordering_respects_aliasing
@@ -5312,18 +5284,14 @@ numba.tests.test_target_extension.TestTargetOffload.test_basic_offload
 numba.tests.test_target_overloadselector.TestAmbiguousOverloads.test_ambiguous_casts
 numba.tests.test_target_overloadselector.TestAmbiguousOverloads.test_ambiguous_functions
 numba.tests.test_target_overloadselector.TestOverloadSelector.test_ambiguous_detection
-numba.tests.test_target_overloadselector.TestOverloadSelector.test_cache
 numba.tests.test_target_overloadselector.TestOverloadSelector.test_match
 numba.tests.test_target_overloadselector.TestOverloadSelector.test_select_and_sort_1
 numba.tests.test_target_overloadselector.TestOverloadSelector.test_select_and_sort_2
 numba.tests.test_target_overloadselector.TestOverloadSelector.test_subclass_specialization
 numba.tests.test_threadsafety.TestThreadSafety.test_concurrent_guvectorize
-numba.tests.test_threadsafety.TestThreadSafety.test_concurrent_guvectorize_cache
 numba.tests.test_threadsafety.TestThreadSafety.test_concurrent_jit
-numba.tests.test_threadsafety.TestThreadSafety.test_concurrent_jit_cache
 numba.tests.test_threadsafety.TestThreadSafety.test_concurrent_mix_use
 numba.tests.test_threadsafety.TestThreadSafety.test_concurrent_vectorize
-numba.tests.test_threadsafety.TestThreadSafety.test_concurrent_vectorize_cache
 numba.tests.test_tracing.TestTracing.test_class_method
 numba.tests.test_tracing.TestTracing.test_function
 numba.tests.test_tracing.TestTracing.test_injected
@@ -6129,110 +6097,6 @@ numba.tests.test_num_threads.TestNumThreadsBackends.test_threadmask_across_fork_
 numba.tests.test_num_threads.TestNumThreadsBackends.test_threadmask_across_fork_workqueue_2_threads
 numba.tests.test_num_threads.TestNumThreadsBackends.test_threadmask_across_fork_workqueue_4_threads
 numba.tests.test_num_threads.TestNumThreadsBackends.test_threadmask_across_fork_workqueue_8_threads
-numba.tests.test_parallel_backend.TestForkSafetyIssues.test_check_threading_layer_is_gnu
-numba.tests.test_parallel_backend.TestForkSafetyIssues.test_par_parent_explicit_mp_fork_par_child
-numba.tests.test_parallel_backend.TestForkSafetyIssues.test_par_parent_implicit_mp_fork_par_child
-numba.tests.test_parallel_backend.TestForkSafetyIssues.test_par_parent_mp_spawn_par_child_par_parent
-numba.tests.test_parallel_backend.TestForkSafetyIssues.test_par_parent_os_fork_par_child
-numba.tests.test_parallel_backend.TestForkSafetyIssues.test_serial_parent_explicit_mp_fork_par_child_then_par_parent
-numba.tests.test_parallel_backend.TestForkSafetyIssues.test_serial_parent_implicit_mp_fork_par_child_then_par_parent
-numba.tests.test_parallel_backend.TestInitSafetyIssues.test_lazy_lock_init
-numba.tests.test_parallel_backend.TestInitSafetyIssues.test_orphaned_semaphore
-numba.tests.test_parallel_backend.TestMiscBackendIssues.test_omp_stack_overflow
-numba.tests.test_parallel_backend.TestMiscBackendIssues.test_single_thread_tbb
-numba.tests.test_parallel_backend.TestMiscBackendIssues.test_workqueue_aborts_on_nested_parallelism
-numba.tests.test_parallel_backend.TestMiscBackendIssues.test_workqueue_handles_fork_from_non_main_thread
-numba.tests.test_parallel_backend.TestOpenMPVendors.test_vendors
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_fork_concurrent_guvectorize
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_fork_concurrent_jit
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_fork_concurrent_mix_use
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_fork_concurrent_mix_use_masks
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_fork_concurrent_vectorize
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_forkserver_concurrent_guvectorize
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_forkserver_concurrent_jit
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_forkserver_concurrent_mix_use
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_forkserver_concurrent_mix_use_masks
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_forkserver_concurrent_vectorize
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_spawn_concurrent_guvectorize
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_spawn_concurrent_jit
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_spawn_concurrent_mix_use
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_spawn_concurrent_mix_use_masks
-numba.tests.test_parallel_backend.TestParallelBackend.test_multiprocessing_spawn_concurrent_vectorize
-numba.tests.test_parallel_backend.TestParallelBackend.test_random_concurrent_guvectorize
-numba.tests.test_parallel_backend.TestParallelBackend.test_random_concurrent_jit
-numba.tests.test_parallel_backend.TestParallelBackend.test_random_concurrent_mix_use
-numba.tests.test_parallel_backend.TestParallelBackend.test_random_concurrent_mix_use_masks
-numba.tests.test_parallel_backend.TestParallelBackend.test_random_concurrent_vectorize
-numba.tests.test_parallel_backend.TestParallelBackend.test_threading_concurrent_guvectorize
-numba.tests.test_parallel_backend.TestParallelBackend.test_threading_concurrent_jit
-numba.tests.test_parallel_backend.TestParallelBackend.test_threading_concurrent_mix_use
-numba.tests.test_parallel_backend.TestParallelBackend.test_threading_concurrent_mix_use_masks
-numba.tests.test_parallel_backend.TestParallelBackend.test_threading_concurrent_vectorize
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_fork_concurrent_guvectorize_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_fork_concurrent_guvectorize_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_fork_concurrent_jit_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_fork_concurrent_jit_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_fork_concurrent_mix_use_masks_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_fork_concurrent_mix_use_masks_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_fork_concurrent_mix_use_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_fork_concurrent_mix_use_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_fork_concurrent_vectorize_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_fork_concurrent_vectorize_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_guvectorize_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_guvectorize_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_guvectorize_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_jit_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_jit_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_jit_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_mix_use_masks_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_mix_use_masks_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_mix_use_masks_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_mix_use_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_mix_use_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_mix_use_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_vectorize_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_vectorize_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_forkserver_concurrent_vectorize_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_guvectorize_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_guvectorize_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_guvectorize_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_jit_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_jit_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_jit_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_mix_use_masks_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_mix_use_masks_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_mix_use_masks_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_mix_use_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_mix_use_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_mix_use_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_vectorize_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_vectorize_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_multiprocessing_spawn_concurrent_vectorize_workqueue
-numba.tests.test_parallel_backend.TestSpecificBackend.test_random_concurrent_guvectorize_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_random_concurrent_jit_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_random_concurrent_mix_use_masks_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_random_concurrent_mix_use_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_random_concurrent_vectorize_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_threading_concurrent_guvectorize_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_threading_concurrent_guvectorize_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_threading_concurrent_jit_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_threading_concurrent_jit_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_threading_concurrent_mix_use_masks_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_threading_concurrent_mix_use_masks_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_threading_concurrent_mix_use_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_threading_concurrent_mix_use_tbb
-numba.tests.test_parallel_backend.TestSpecificBackend.test_threading_concurrent_vectorize_omp
-numba.tests.test_parallel_backend.TestSpecificBackend.test_threading_concurrent_vectorize_tbb
-numba.tests.test_parallel_backend.TestTBBSpecificIssues.test_fork_from_non_main_thread
-numba.tests.test_parallel_backend.TestTBBSpecificIssues.test_lifetime_of_task_scheduler_handle
-numba.tests.test_parallel_backend.TestThreadingLayerPriority.test_invalid_env_var
-numba.tests.test_parallel_backend.TestThreadingLayerPriority.test_omp
-numba.tests.test_parallel_backend.TestThreadingLayerPriority.test_tbb
-numba.tests.test_parallel_backend.TestThreadingLayerPriority.test_valid_env_var
-numba.tests.test_parallel_backend.TestThreadingLayerPriority.test_workqueue
-numba.tests.test_parallel_backend.TestThreadingLayerSelection.test_threading_layer_selector_omp
-numba.tests.test_parallel_backend.TestThreadingLayerSelection.test_threading_layer_selector_tbb
-numba.tests.test_parallel_backend.TestThreadingLayerSelection.test_threading_layer_selector_workqueue
 numba.tests.test_svml.TestSVML.test_scalar_context_asm
 numba.tests.test_svml.TestSVML.test_scalar_context_result
 numba.tests.test_svml.TestSVML.test_svml_asm
-- 
2.52.0

